docs/platform/webcontainers/project-config.md

```diff
@@ -121,6 +121,29 @@ The default `"auto"` setting enables showing changes as user types, which may pr
 
 A map of default environment variables that will be set in each top-level shell process.
 
+### <var>view</var>
+
+<table>
+  <tr>
+    <td>Type</td>
+    <td><code>"default" | "editor"</code></td>
+  </tr>
+  <tr>
+    <td>Default</td>
+    <td><code>"default"</code></td>
+  </tr>
+</table>
+
+The `view` option allows you to configure the layout the editor uses at startup.
+
+The available settings are:
+
+- `"default"` (default): Editor & Preview side-by-side
+- `"editor"`: Just the editor, no preview
```

Comment:
Nitpick: editor and preview are capitalized above, but not here

Reply:
changed